packages/babel-plugin-proposal-regexp-modifiers/src/index.ts

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable @babel/development/plugin-name */
+import { createRegExpFeaturePlugin } from "@babel/helper-create-regexp-features-plugin";
+import { declare } from "@babel/helper-plugin-utils";
+
+export default declare(api => {
+  api.assertVersion("^7.19.0");
```

Comment:
This line of code was copied from another plugin, I'm not sure what it does.😞

Reply:
This plugin doesn't technically require `@babel/core` 7.19.0, but hopefully this will encourage people to upgrade :wink: